deprecation/README.md

```diff
@@ -2,4 +2,17 @@
 
 * `Deprecated` PyPI package is used for deprecation
 * annotate deprecated functions with `@depracted` decorator, optionally with message
-* functions will be removed after 6 months from the release where they were marked as deprecated
+* functions will be removed after 6 months from the release in which they were marked as deprecated
```

Comment:
The number of months is your suggestion or the info from `Deprecated` module?

Reply:
I looked into few projects that have explicit policy...
However thinking about it, it should be more specific, since it could mean deprecating function in 0.X.Y version and then removing it right in the next version after Y.